Content.Client/UserInterface/Systems/Scoreboard/ScoreboardUIController.cs

```diff
@@ -0,0 +1,64 @@
+using Content.Client.RoundEnd;
+using Robust.Client.Input;
+using Content.Shared.Input;
+using Robust.Shared.Input.Binding;
+using Content.Client.GameTicking.Managers;
+using Content.Client.Lobby;
+using Content.Client.Gameplay;
+using Robust.Shared.Log;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controllers;
+
+namespace Content.Client.UserInterface.Scoreboard;
```

Comment:
```suggestion
namespace Content.Client.UserInterface.Systems.Scoreboard;
```
To match the file structure.